File: RSCDevStats/Models/SheetsModels/PlayerDetails.cs
using System;
using System.Collections.Generic;

namespace RSCWebApi
{
    public class PlayerDetails
    {
        public string RSCId { get; set; }
        public string Name { get; set; }
        public List<PlayerTrackerLink> PlayerTrackerLinks { get; set; }
        public string Franchise { get; set; }
        public string Team { get; set; }
        public int BaseMmr { get; set; }
        public int CurrentMmr { get; set; }
        public string ContractStatus { get; set; }
        public int ContractLength { get; set; }
    }

    public class PlayerTrackerLink
    {
        public PlayerTrackerLink(string trackerLink, string platform, string platformId)
        {
            TrackerLink = trackerLink;
            Platform = platform;
            PlatformId = platformId;
        }
        public string TrackerLink { get; set; }
        public string Platform { get; set; }
        public string PlatformId { get; set; }
    }
}

File: RSCDevStats/ClientApp/src/models/PlayerDetails.ts
export interface PlayerDetails {
	RSCId: string
	Name: string
	PlayerTrackerLinks: PlayerTrackerLink[]
	Franchise: string
	Team: string
	BaseMmr: number
	CurrentMmr: number
	ContractStatus: string
	ContractLength: string
}

export interface PlayerTrackerLink {
	TrackerLink: string
	Platform: string
	PlatformId: string
}
